pkg/operator/webhookconfig.go

```diff
@@ -14,24 +13,26 @@ import (
 	"sigs.k8s.io/controller-runtime/pkg/manager"
 )
 
-// WebhookConfigurationName is the name of the webhook configurations to be
-// updated with the current CA certificate.
+// WebhookConfigurationName is the name of the webhook configuration.
 const WebhookConfigurationName = "autoscaling.openshift.io"
 
+// InjectCABundleAnnotationName is the annotation used by the
+// service-ca-operator to indicate which resources it should inject the CA into.
+const InjectCABundleAnnotationName = "service.beta.openshift.io/inject-cabundle"
```

Comment:
Pulling in an entire new dependency for one constant, and still having to take manual action to update it if it changes doesn't seem worth it to me.

Reply:
+1 to just defining it here. Also, since the name of the annotation is supported for customers, it is effectively a stable API and any changes would follow a reasonable deprecation cycle.